tests/Unit/UpgradePostgresScriptTest.php

```
@@ -62,6 +62,25 @@ function assertBashSyntaxIsValid(string $path): void
     'nightly upgrade' => 'other/nightly/upgrade.sh',
 ]);
 
+it('persists the target image and runtime version before recreating containers', function (string $path) {
+    $script = file_get_contents(getcwd().'/'.$path);
+
+    $latestImagePosition = strpos($script, 'set_env_var "LATEST_IMAGE" "$LATEST_IMAGE"');
```

PHPStan (2.2.7)

[warning] Parameter `#1` $haystack of function strpos expects string, string|false given.


(argument.type)